src/store/plugins/fungibleTokens.js

```diff
@@ -158,6 +159,29 @@ export default (store) => {
           decodedResult !== undefined ? 'change_allowance' : 'create_allowance'
         ](activeNetwork.tipContractV2.replace('ct_', 'ak_'), allowanceAmount);
       },
+      async getContractTokenPairs(
```

Comment:
This function is updated in other PR https://github.com/aeternity/superhero-wallet/pull/1493.